src/Authentication/Authentication/Models/AuthContext.cs

```diff
@@ -1,16 +1,12 @@
 ﻿// ------------------------------------------------------------------------------
 //  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
 // ------------------------------------------------------------------------------
-namespace Microsoft.Graph.PowerShell.Authentication.Models
+namespace Microsoft.Graph.PowerShell.Authentication
 {
-    public enum AuthenticationType
-    {
-        Delegated,
-        AppOnly
-    }
-
-    public class AuthConfig
+    using Newtonsoft.Json;
```

Comment:
>using Newtonsoft.Json; [](start = 4, length = 22)

since we're doing the work to convert these in the core library do we have an issue for updating these?

Reply:
The PS SDK still uses the old core library. We will only need to serialize this class when we start storing auth context locally. I'll remove the attribute for now.